tests/phpunit/tests/user.php

```diff
@@ -2185,4 +2185,67 @@ public function export_additional_user_profile_data_with_dup_name() {
 
 		return $additional_profile_data;
 	}
+
+	/**
+	 * Test if the use_ssl is not written unneccesarily.
```

Comment:
it would be great if the test actually showed that there was no db write (if that is possible to determine here)

Reply:
I have updated the comment.